isaric/parsers/datcov-southafrica.toml

```diff
@@ -650,12 +653,13 @@
   is_present = { field = "flw_headache", values = { 0 = false, 1 = true } }
   if.any = [{ flw_headache = 2 }, { flw_headache = 4 }]
 
-  # [[observation]]
-  #   name = "headache"
-  #   phase = "followup"
-  #   date = { ref = "followupDateHierarchy" }
-  #   is_present = { field = "flw2_headache", values = { 0 = false, 1 = true } }
-  #   if.any = [{ flw2_headache = 2 }, { flw2_headache = 4 }]
+[[observation]]
+  name = "headache"
+  phase = "followup"
+  date = { field = "flw2_survey_date_{n}" }
+  is_present = { field = "flw2_headache_{n}", values = { 0 = false, 1 = true } }
+  if.any = [{ "flw2_headache_{n}" = 1 }, { "flw2_headache_{n}" = 0 }]
```

Comment:
How does this part work? can't figure it out ... also applies for the other mappings above